raiden/network/proxies/secret_registry.py

```diff
@@ -44,10 +44,12 @@ def __init__(
             CONTRACT_MANAGER.get_contract_abi(CONTRACT_SECRET_REGISTRY),
             to_normalized_address(secret_registry_address),
         )
-        CONTRACT_MANAGER.check_contract_version(
-            proxy.functions.contract_version().call(),
-            CONTRACT_SECRET_REGISTRY
-        )
+
+        # TODO: add this back
```

Comment:
If you leave a TODO like that please make an issue and assign yourself (or someone else). Same goes for the other instances below

Reply:
Don't think that's worth an issue right now, added a task here: https://github.com/raiden-network/raiden/issues/1383

Reply:
@palango Whatever floats your boat. It's just to keep track of them. If it's part of an issue adding a task is fine. Thank you!